docs~/docs/reference/global-collider.md

```diff
@@ -0,0 +1,27 @@
+﻿# Global Collider
+
+The `MA Global Collider` component is used to define colliders on your avatar which can globally interact with other dynamic systems.
+
+![Global Collider](global-collider.png)
+
+## When should I use it?
+
+You can place this component on any game object and define its shape like a standard capsule or VRChat Physbone collider.
+
+## What does it do?
+
+The functionality of this component varies by platform.
```

Comment:
We should call out:
1) Currently only vrchat is supported
2) On VRChat, we steal finger bones, so you might run out of global colliders eventually
3) On other platforms, we may implement this in a different way

And probably document the weird contact-only values?